src/amo/components/SearchResult/index.js

```diff
@@ -33,8 +34,21 @@ export class SearchResultBase extends React.Component {
     return addon && addon.type === ADDON_TYPE_THEME;
   }
 
+  lastUpdatedText() {
+    const { addon, i18n, sortedByDate } = this.props;
+    const lastUpdated = addon && sortedByDate && i18n.sprintf(
+      // translators: This will output, in English:
+      // "2 months ago (Dec 12 2016)"
+      i18n.gettext('%(timeFromNow)s (%(date)s)'), {
+        timeFromNow: i18n.moment(addon.last_updated).fromNow(),
+        date: i18n.moment(addon.last_updated).format('ll'),
+      }
+    );
+    return lastUpdated && `${i18n.gettext('Last updated')}: ${lastUpdated}`;
```

Comment:
You may want to wait for @pwalm to weight in but I think this is cramming a lot of data into a small space. I would suggest doing something like *Updated 2 days ago* and then including the full date string in a tooltip.